### PR TITLE
add purchase

### PR DESCRIPTION
**Pull request by sguan12**
Added edit text fields for users to input information about transaction, created new Transaction class, when "Submit" button is clicked the transaction information is stored in Parse database

### CODE REVIEW

File: app/src/main/java/com/example/coinage/fragments/AddFragment.java [hunk 1]
```diff
@@ -1,20 +1,45 @@
...
+import java.text.ParseException;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
 
 public class AddFragment extends Fragment {
     public static final String TAG = "AddFragment";
 
+    final Calendar myCalendar= Calendar.getInstance();
+    String myFormat="MM/dd/yy";
```
Comment (chetfb) [inline app/src/main/java/com/example/coinage/fragments/AddFragment.java]: static final?

**Review by chetfb — APPROVED** [review]
It may be a good idea to screen capture the final view and attach to the PR request in the PR description. (this is how we show that we have tested and the feature is working in Meta PRs)

**Comment by sguan12** [comment]
image

image

File: app/src/main/java/com/example/coinage/fragments/AddFragment.java [hunk 2]
```diff
@@ -31,9 +31,9 @@
 public class AddFragment extends Fragment {
     public static final String TAG = "AddFragment";
 
-    final Calendar myCalendar= Calendar.getInstance();
-    String myFormat="MM/dd/yy";
-    SimpleDateFormat dateFormat = new SimpleDateFormat(myFormat, Locale.US);
+    public static final Calendar myCalendar = Calendar.getInstance();
```
Comment (chetfb) [inline app/src/main/java/com/example/coinage/fragments/AddFragment.java:34]: Sorry, only the constants need to be public static. Others  need not be static -- just public/private final.

**Comment by wgregmeta** [comment]
screenshots look great! A video of the flow is even better, are you able to download QuickTime for your work laptop on the MSC? It's a really easy way to record your screen

**Pull request merged**